### PR TITLE
[NUXE-55] Updated styles to button toolbar kit

### PR DESCRIPTION
**Pull request by christinaatai**
#### Screens

Screen Shot 2020-10-08 at 3 39 57 PM
Screen Shot 2020-10-08 at 3 40 06 PM
Screen Shot 2020-10-08 at 3 39 44 PM
Screen Shot 2020-10-08 at 3 39 37 PM

#### Breaking Changes

No, this kit is not being used in Nitro yet.

#### Runway Ticket URL

https://nitro.powerhrg.com/runway/backlog_items/NUXE-55

#### How to test this

[INSERT TESTING DETAILS]

#### Checklist:

- [x] **LABELS** Add a label: `enhancement`, `bug`, `improvement`, `new kit`, `depreciation`, or `breaking`. See [Changelog & Labels](https://github.com/powerhome/playbook/wiki/Changelog-&-Labels) for details.
- [x] **URGENCY** Please select a release milestone
- [x] **DEPLOY** Please add the `Milano` label when you are ready for a review.
- [x] **SCREENSHOT** Please add a screen shot or two.
- [x] **SPECS** Please cover your changes with specs.
- [ ] **READ DOCS** Please make sure you have read and understand the [Playbook Release Process](https://github.com/powerhome/playbook/wiki/Playbook-Releases)

*The normal release cut off deadline is 3p EDT each week. Please reach out to the release team if you have an urgent request or need a release off cycle.*

### CODE REVIEW

File: app/pb_kits/playbook/pb_button_toolbar/_button_toolbar.scss [hunk 1]
```diff
@@ -1,6 +1,39 @@
...
+    background-color:rgba($bg_dark, $opacity);
+  }
+  content: '';
+  position: relative;
+  width: 2px;
+  height: $space_md;
+  top: 0;
+  bottom: 0;
+  right: calc(#{$space_lg} + 2px);
+}
```
Comment (jasoncypret) [inline app/pb_kits/playbook/pb_button_toolbar/_button_toolbar.scss:36]: These could possibly be combined into 1 mixin

File: app/pb_kits/playbook/pb_button_toolbar/_button_toolbar.scss [hunk 2]
```diff
@@ -73,37 +109,164 @@
...
+              border-bottom-right-radius: 0;
+              border-top-right-radius: 0;
+              border-right-width: 0;
+            }
+            &:not(:first-child){
+              border-bottom-left-radius: 0;
+              border-top-left-radius: 0;
+              border-left-width: 0;
+            }
+          }
```
Comment (jasoncypret) [inline app/pb_kits/playbook/pb_button_toolbar/_button_toolbar.scss:133]: This seems to be the same as lines 145-155 can you eliminate one by moving it to the parent class?

File: app/pb_kits/playbook/pb_button_toolbar/_button_toolbar.scss [hunk 3]
```diff
@@ -73,37 +109,164 @@
...
+              border-bottom-right-radius: 0;
+              border-bottom-left-radius: 0;
+              border-bottom-width: 0;
+            }
+            &:not(:first-child){
+              border-top-right-radius: 0;
+              border-top-left-radius: 0;
+              border-top-width: 0;
+            }
+          }
```
Comment (jasoncypret) [inline app/pb_kits/playbook/pb_button_toolbar/_button_toolbar.scss:182]: Same...seems like we can eliminate

File: app/pb_kits/playbook/pb_button_toolbar/_button_toolbar.scss (hunk 1 above)
Comment (jasoncypret) [reply app/pb_kits/playbook/pb_button_toolbar/_button_toolbar.scss:36]: Another thought is moved to a separate mixin file to make this a bit more readable

File: app/pb_kits/playbook/pb_button_toolbar/_button_toolbar.scss [hunk 4]
```diff
@@ -73,37 +109,164 @@
...
-  }
\ No newline at end of file
+  }
+
+// Dark =============
+
+[class^=pb_button_toolbar]{
+  &[class*=_connected] {
+    &[class*=_vertical] {
+      &[class*=_primary].dark {
```
Comment (jasoncypret) [inline app/pb_kits/playbook/pb_button_toolbar/_button_toolbar.scss:216]: Consider making a separate file or mixin

File: app/pb_kits/playbook/pb_button_toolbar/docs/_button_toolbar_default.jsx [hunk 5]
```diff
@@ -1,25 +1,58 @@
...
+          text="Event"
+      />
+      <Button
+          {...props}
+          text="Training"
+      />
+      <Button
+          {...props}
+          text="Not Working"
+      />
```
Comment (jasoncypret) [inline app/pb_kits/playbook/pb_button_toolbar/docs/_button_toolbar_default.jsx:30]: Can we think of other examples for the button text? These are very power specific, and also I think we can get creative here. See the nav examples for inspiration

**Review by jasoncypret — CHANGES_REQUESTED** [review]
I'd like to improve some of the CSS & SASS.

Looking at other design systems, I think we need to not allow these buttons to float to two lines. `white-space: nowrap;`

The readability and length of this file concern me. I'd recommend possibly breaking up the file into horizontal and vertical variants, removing duplicated CSS (like the button hovers for example).

I'd like to go through the whole file and look for all the duplications. See if they can be eliminated in a smart way

**Review by jasperfurniss — APPROVED** [review]

**Review by jasoncypret — APPROVED** [review]

**Pull request merged**